app/views/components/gds/exportsInputText.scala.html

```diff
@@ -26,41 +30,45 @@
     hintKey: Option[String] = None,
     hintKeyLinks: Seq[Html] = Seq.empty,
     hintHtml: Option[Html] = None,
+    bodyHtml: Option[Html] = None,
```

Comment:
Could this be renamed to something like `bodyUnderLabel`, `paragraphUnderLabel` or similar so to make clear what's for?

Reply:
Don't really want to include positional information in a param name (as it might not always be in that position). I think the concept of each field component having a form, label and then an optional body and hint element is clear enough no? Could rename 'body' to be 'description' I suppose if that is any better for you?